src/test/nameservice.ts

```diff
@@ -97,6 +118,35 @@ describe('BlockstackService tests', () => {
     })
     it('given identityClaim with mnemonic with invalid cruxID, should throw "CruxIdLengthValidation" | "CruxIdNamespaceValidation"')
     it('given identityClaim with mnemonic and non-corresponding cruxID, should throw error')
+    
+    it('given identityClaim without secret', async() => {
```

Comment:
isn't this the same case as `given cruxID without identityClaim, should throw "CouldNotFindMnemonicToRestoreIdentity"`? also please make sure your `it` statements are very descriptive of the case.